src/components/TheDashboardFooter.vue

```diff
@@ -72,21 +74,18 @@
     </div>
 
     <money-transfer-modal
-      v-if="showTransferModal"
-      @close="
-        ;(showTransferModal = false), $store.commit('setModalState', false)
-      "
+      v-if="getCurrentModal?.component == 'MoneyTransferModal'"
```

Comment:
Hmm, sometimes it seems the check is in the modal component, and here it is outside. Not sure which is best, no way to use only one way ?